concrete/src/Support/i18n.php

```diff
@@ -8,3 +8,11 @@
 
 // To translate the value of the VERSION_INITIAL_COMMENT constant
 t('Initial Version');
+
+// To translate the names of the autentication types
+// @see \Concrete\Core\Package\StartingPointPackage::add_users
+tc('AuthenticationType', 'Standard');
```

Comment:
> Shouldn't these be handled dynamically as with other objects that have translatable names?

Sure they are: see https://github.com/mlocati/concrete5-translation-library/commit/fe16898feb6f60bb9848c98de794734f3c40f1de

BTW we need to tell the translation system that we have [these 5 predefined authentication types](https://github.com/concrete5/concrete5/blob/10e6b86af2c6022bd83d553271301943a416c6ce/concrete/src/Package/StartingPointPackage.php#L376-L380), so that their names can be sent to https://translate.concrete5.org/ and be translated (the final result is that their translation are included in the core language files).